site/docs/tools/iceberg/spark.md

```diff
@@ -320,30 +185,22 @@ supports the Nessie Iceberg Catalog also supports.
 
 ## Reading
 
-Reading is more straightforward between spark 2 and spark 3. We will look at both versions together in this section. To
-read a Nessie table in iceberg simply:
+To read a Nessie table in iceberg simply:
 
 === "Java"
     ``` java
-    // Spark2:
-    regionDf = spark.read().format("iceberg")
-        .load("nessie.testing.region");
-
-    // Spark3:
     regionDf = spark.table("nessie.testing.region");
     ```
 === "Python"
     ``` python
-    # same code as above to create the testing.region table
     region_df = spark.read.format("iceberg").load("nessie.testing.region")
     ```
 === "SQL"
     ``` sql
-    -- Spark3 only
     SELECT * FROM nessie.testing.city
     ```
     ``` sql
-    -- Spark3 only, read from the `etl` branch
```

Comment:
we might also want to remove the mentions from

1) https://github.com/projectnessie/nessie/blob/a3837e1d21c541fe48e2d131e65fcc49b91fa873/site/docs/tools/iceberg/index.md?plain=1#L3

2) https://github.com/projectnessie/nessie/blob/a3837e1d21c541fe48e2d131e65fcc49b91fa873/site/docs/tools/index.md?plain=1#L23

3) https://github.com/projectnessie/nessie/blob/a3837e1d21c541fe48e2d131e65fcc49b91fa873/site/docs/tools/index.md?plain=1#L34